jupyterlab/handlers/yjs_echo_ws.py

```diff
@@ -13,6 +13,7 @@
 releaseLockMessageType = 126
 requestInitializedContentMessageType = 125
 putInitializedContentMessageType = 124
+renameSession = 123
```

Comment:
I would suggest renaming it for consistency:

```suggestion
renameSessionMessageType = 123
```

Is there a reason why those message type are not defined as an [`IntEnum`](https://docs.python.org/3/library/enum.html#enum.IntEnum)?

Reply:
Thanks! I implemented your suggestion.

I switched to IntEnum. The only reason why I didn't use this before was that I didn't know about it ¯\_(ツ)_/¯ (I'm a Python newbie).

Reply:
Thanks for making the changes @dmonad